### PR TITLE
use stencil store and new editor

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

**Review by MikeTschudi — DISMISSED** [review]
Splendid accomplishment--thank you!

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "generate": "stencil generate"
   },
   "devDependencies": {
+    "@stencil/store": "^1.4.1",
```
[the remark this answers is not in the document]
Comment (jmhauck) [reply package.json:26]: thanks...it is indeed...moved.

File: src/components/solution-configuration/solution-configuration.tsx [hunk 2]
```diff
@@ -39,14 +40,16 @@ export class SolutionConfiguration {
   //  Host element access
   //
   //--------------------------------------------------------------------------
-  @Element() el: HTMLElement;
```
[the remark this answers is not in the document]
Comment (jmhauck) [reply src/components/solution-configuration/solution-configuration.tsx:42]: Was looking in calcite and they use the auto generated types.

[the remark this answers is not in the document]
Comment (jmhauck) [reply src/components/solution-configuration/solution-configuration.tsx:42]: I'll update and submit a PR

**Comment by jmhauck** [comment]
Thanks @MikeTschudi I went ahead and added @Element statements for each of the components for consistency. Let me know what you think.

**Review by MikeTschudi — APPROVED** [review]
Thanks, John!

**Pull request merged**